content/spire/try/getting-started-linux-macos-x.md

```diff
@@ -32,7 +32,7 @@ The commands in this getting started guide can be run as a standard user or root
 ## Downloading SPIRE for Linux
 
 ```
-$ curl -s -N -L https://github.com/spiffe/spire/releases/download/0.9.0/spire-0.9.0-linux-x86_64-glibc.tar.gz | tar xz
+$ curl -s -N -L https://github.com/spiffe/spire/releases/download/0.10.0/spire-0.10.0-linux-x86_64-glibc.tar.gz | tar xz
 ```
 
 This will create a `spire-0.9.0` directory containing the binaries and example configuration files.
```

Comment:
Please change to:

This will create a `spire-{{< spire-latest "version" >}}` directory...

Also, under "Building SPIRE on macOS/Darwin", I think "--branch v0.8" means SPIRE 0.8. Can you confirm this should be v0.10 and make the change if so? I'm not familiar with that git clone syntax.

Reply:
`--branch v0.8` can be replaced with `--branch {{< spire-latest "tag" >}}` which points, as you have guessed, to the latest tag in the spire repo